client/etcd/options.go

```diff
@@ -35,14 +35,14 @@ type Options interface {
 	Zone() string
 	SetZone(z string) Options
 
-	ServiceInitTimeout() time.Duration
-	SetServiceInitTimeout(timeout time.Duration) Options
+	Service() string
```

Comment:
Ah. Looks like this interface exposes `Zone`, `Env` and `Name`; instead of that, could it have a ServiceID field?